website/src/components/AmbassadorGrid/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { createAmbassadorCard } from "../AmbassadorCard";
+import ambassadors from "../../pages/ambassadors.json";
+
+const AmbassadorGrid = () => (
+  <div
+    style={{
+      display: "grid",
+      gridTemplateColumns: "repeat(auto-fit, minmax(320px, 1fr))",
+      gap: "20px",
+    }}
+  >
+    {ambassadors.map((ambassador, index) => (
+      <React.Fragment key={index}>
+        {createAmbassadorCard({
+          title: ambassador.title,
+          role: ambassador.role,
+          company: ambassador.company,
+          companyLink: ambassador.companyLink,
+          personalLink: ambassador.personalLink,
+          img: ambassador.img,
+        })}
+      </React.Fragment>
```

Comment:
the `React.Fragment` is unnecessary here—an array of elements can just be included as a direct child of a component: https://react.dev/learn/rendering-lists

```suggestion
        {createAmbassadorCard({
          title: ambassador.title,
          role: ambassador.role,
          company: ambassador.company,
          companyLink: ambassador.companyLink,
          personalLink: ambassador.personalLink,
          img: ambassador.img,
        })}
```

Reply:
Ahhh I thought I had to swallow the index since I was getting a TS typing error earlier. Great to know that if I had just used the component directly I wouldn't need this :)

Reply:
ahh was it this error? https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key

generally if you're rendering components with a `.map()` or something, you have to specify a `key` prop that uniquely identifies the element being rendered (e.g. `<AmbassadorCard key={ambassador.title} {...ambassador} />`). things will still generally render okay without it if the list is static, but it's good to include anyway.

this can be added in a follow-up PR!